File: 2119-a-number-after-a-double-reversal/2119-a-number-after-a-double-reversal.java
class Solution {
    public boolean isSameAfterReversals(int num) {
        int original = num;
        int rev = 0;
        int digit = 0;
        int num2 = 0;
        while(num > 0){
            digit = num%10;
            rev = rev * 10 + digit;
            num/=10;
            num2 = num;
        }
        int rev2 = 0;
        while(rev > 0){
            digit = rev%10;
            rev2 = rev2 * 10 + digit;
            rev/=10;
        }
        if(original==rev2){
            return true;
        }
        else{
            return false;
        }
    }
}

File: 2119-a-number-after-a-double-reversal/2119-a-number-after-a-double-reversal.py
class Solution(object):
    def isSameAfterReversals(self, num):
        #rev1 = list(str(num))
        #rev1.reverse()
        #rev1 = int(''.join(rev1))

        #rev2 = list(str(rev1))
        #rev2.reverse()
        #rev2 = int(''.join(rev2))
        #return rev2==num

        #2nd var

        return num == int(str(int(str(num)[::-1]))[::-1])



